magefiles/magefile.go

```diff
@@ -42,7 +42,8 @@ var (
 	jobName          = utils.GetEnv("JOB_NAME", "")
 	// can be periodic, presubmit or postsubmit
 	jobType                    = utils.GetEnv("JOB_TYPE", "")
-	reposToDeleteDefaultRegexp = "jvm-build-suite|e2e-dotnet|build-suite-test|e2e-multiple-components|e2e-nodejs|pet-clinic-e2e|test-app|multi-component-application|e2e-quayio|petclinic"
+	reposToDeleteDefaultRegexp = "jvm-build|e2e-dotnet|build-suite|e2e|pet-clinic-e2e|test-app|multi-component-application|e2e-quayio|petclinic|test-app|integ-app|^dockerfile-|new-|^python|my-app|^test-|^multi-component"
```

Comment:
Since we have `^multi-component`, I think we can remove `multi-component-application` in order to `reposToDeleteDefaultRegexp` be shorter

```suggestion
	reposToDeleteDefaultRegexp = "jvm-build|e2e-dotnet|build-suite|e2e|pet-clinic-e2e|test-app|e2e-quayio|petclinic|test-app|integ-app|^dockerfile-|new-|^python|my-app|^test-|^multi-component"
```